packages/manager/modules/pci/src/projects/project/storages/databases/database/general-information/general-information.html

```diff
@@ -15,7 +15,7 @@
     </oui-message>
 
     <oui-message
-        data-ng-if="$ctrl.users.length === 0 && $ctrl.allowedIps.length > 0"
+        data-ng-if="$ctrl.users.length === 0 && $ctrl.isFeatureActivated('usersTab') && $ctrl.allowedIps.length > 0"
```

Comment:
I think it's better to check first the feature availability and then the rest to avoid useless computation.
```suggestion
        data-ng-if="$ctrl.isFeatureActivated('usersTab') && $ctrl.users.length === 0 && $ctrl.allowedIps.length > 0"
```

Reply:
Those warning messages have been reworked here: https://github.com/ovh/manager/pull/5872